problem_builder/mentoring.py

```diff
@@ -918,11 +1022,37 @@ def allowed_nested_blocks(self):
     def update_active_step(self, new_value, suffix=''):
         if new_value < len(self.steps):
             self.active_step = new_value
+        elif new_value == len(self.steps):
+            if self.has_review_step:
+                self.active_step = -1
         return {
             'active_step': self.active_step
         }
 
     @XBlock.json_handler
+    def update_num_attempts(self, data, suffix=''):
+        if self.num_attempts < self.max_attempts:
+            self.num_attempts += 1
+        return {
+            'num_attempts': self.num_attempts
+        }
+
+    @XBlock.json_handler
+    def get_grade(self, data, suffix):
+        score = self.score
+        return {
+            'score': score.percentage,
+            'correct_answers': len(score.correct),
+            'incorrect_answers': len(score.incorrect),
+            'partially_correct_answers': len(score.partially_correct),
+            'correct': self.correct_json(stringify=False),
```

Comment:
@itsjeyd Nit: The `stringify=False` parameter is named on the line above (`correct`), but not named on the two lines below (`incorrect`, `partial`), can we make it consistent?